### ISSUE TITLE
Is the "upstream" project page gone?

### ISSUE THREAD

**Issue by petterreinholdtsen** [Rolinh/dfc#34]
This project github description say it is a mirror of https://projects.gw-computing.net/projects/dfc , but I am unable to reach the URL.  Is the upstream project disappeared from the Internet?

**Comment by rolinh** [comment]
Yes, I was hosting a website which I no longer do. I have to update the readme and upload release tarballs here on GitHub.

**Comment by petterreinholdtsen** [comment]
[Robin Hahling]
> Yes, I was hosting a website which I no longer do. I have to update
> the readme and upload release tarballs here on GitHub.

The background for my question is that I have been helping the Debian
maintainer of this package getting the package into Debian, but have
lost contact with him and wondered if this package was no longer being
maintained upstream too, as the home page URL no longer worked.

Good to hear that you will make a new release.  Hope someone will step
up to maintain the Debian package, to get this new release into Debian.

<URL: https://tracker.debian.org/pkg/dfc >

-- 
Happy hacking
Petter Reinholdtsen

**Comment by petterreinholdtsen** [comment]
[Petter Reinholdtsen]
> Good to hear that you will make a new release.  Hope someone will step
> up to maintain the Debian package, to get this new release into
> Debian.

To make it a bit clearer that there is no active maintainer, and did a
few non-maintainer uploads fixing the package metadata, to get it off my
radar for now.

Let me know if you or anyone else is interested in taking over Debian
maintenance.

Note, there is a man page patch fixing a typo in
<URL: https://salsa.debian.org/debian/dfc/-/tree/master/debian/patches >.

-- 
Happy hacking
Petter Reinholdtsen

**Comment by rolinh** [comment]
> Let me know if you or anyone else is interested in taking over Debian
maintenance.

I'm not a Debian (or downstream distro) user myself so no, I'm not interested in taking over Debian maintenance. I'm also not aware of anyone ready to volunteer.

**Comment by petterreinholdtsen** [comment]
[Robin Hahling]
> I'm not a Debian (or downstream distro) user myself so no, I'm not
> interested in taking over Debian maintenance. I'm also not aware of
> anyone ready to volunteer.

No worries.  Unless someone show up to take over, the package in Debian
will end up orphaned and possibly also removed from Debian. :)

<URL: https://tracker.debian.org/pkg/dfc > only show a few popcon users,
so it will most likely not cause any drama to drop it. :)

-- 
Happy hacking
Petter Reinholdtsen
